sql/catalyst/src/main/scala/org/apache/spark/sql/execution/arrow/ArrowWriter.scala

```diff
@@ -485,15 +519,13 @@ private[arrow] class MapWriter(
 
 private[arrow] class NullWriter(val valueVector: NullVector) extends ArrowFieldWriter {
 
-  override def setNull(): Unit = {
-  }
+  override def setNull(): Unit = {}
 
-  override def setValue(input: SpecializedGetters, ordinal: Int): Unit = {
-  }
+  override def setValue(input: SpecializedGetters, ordinal: Int): Unit = {}
 }
 
 private[arrow] class IntervalYearWriter(val valueVector: IntervalYearVector)
-  extends ArrowFieldWriter {
+    extends ArrowFieldWriter {
```

Comment:
did you run scalafmt for the whole code base? Let's don't. It includes too many unncessary changes which make it hard to revert/backport/manage.

Reply:
Let me revert auto format changes and fix format manually.